ga-ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldGAspec.json

```diff
@@ -1948,6 +1948,7 @@
       {
         "UserRoles": [
           "caseworker-civil-solicitor",
+          "caseworker-civil-judge",
```

Comment:
Could you please double check whether we need to add judge for ServiceRequest tab? we should not show the servicerequest tab for judge as per my understanding and confirmed by vasu

@vasudevganesanhmcts  please advise me

Reply:
Removing this would also fix the bug which you have mentioned @vasudevganesanhmcts

Reply:
Even we dont use "casework-civil-judge" role anymore. Kindly avoid adding this role.

Reply:
yup agree . .as explained before we dont use idam caseworker-civil-judge and we use access profiles for judge related roles . . please remove

Reply:
Should I remove it across whole AuthorisationCaseFieldGAspec?
@vasudevganesanhmcts

Reply:
yeah go for it